src/components/widgets/registration/steps/ContactsStep.tsx

```diff
@@ -13,13 +14,31 @@ export const ContactsStep = ({
     onSaveEdits?: () => void;
 }) => {
     const { t } = useTranslation();
-    const { clearErrors } = useFormContext();
+    const { clearErrors, control } = useFormContext();
+
+    const fieldUserNameTelegram = useWatch({
+        control,
+        name: 'userNameTelegram',
+    });
+    const fieldUserNameInstagram = useWatch({
+        control,
+        name: 'userNameInstagram',
+    });
```

Comment:
```suggestion
    const [fieldUserNameTelegram, fieldUserNameInstagram] = useWatch< FormFields , ['userNameTelegram', 'useNameInstagram']>({
        name: ['userNameTelegram', 'useNameInstagram'],
    });
```

also i would call it shorter userNameTelegram and useNameInstagram